### PR TITLE
Removes the separate assets wsgi app and adds a view that uses 'juju-ui' in the url as an indication that it should return a file from the assets path.

### PR DESCRIPTION
**Pull request by jaycee**

### CODE REVIEW

**Comment by jaycee** [comment]
QA:

Load /trusty/juju-gui-10; check the assets that are loaded (e.g. in the network tab in chrome developer tools). Ensure there are no missed assets. Specifically ensure that any url with juju-ui in it worked.

Feel free to pick some files from the assets directory and see if they load with various versions of /some/sort/of/prefix/juju-ui/assets/file/of/your/choice.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2772/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2774/
Test PASSed.

**Comment by kadams54** [comment]
QA Notes:

Browsed around the app a bit with Network Inspector open, looking for error HTTP codes. Verified that the Juju charm details showed up fine. Tried a few assets with oddball prefixes.

QA is OK, :+1: on the code - thanks!

File: jujugui/gui.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 def gui(config):
     options.update(config.registry.settings)
+    config.add_route('jujugui.ui', '/{prefix:.*}juju-ui/{file:.*}')
```
Comment (hatched) [inline jujugui/gui.py]: This is pretty self explanatory but could you add a comment to define what those things in the curlies do?

File: jujugui/static/gui/src/app/index.html [hunk 2]
```diff
@@ -1,440 +0,0 @@
-<!DOCTYPE html>
```
Comment (hatched) [inline jujugui/static/gui/src/app/index.html]: Thanks for this removal driveby

File: jujugui/views.py [hunk 3]
```diff
@@ -2,12 +2,25 @@
...
 
+ASSET_PATH = os.path.join(
+    os.path.dirname(__file__),
+    'static',
+    'gui',
+    'build',
+    'app'
+    )
+
+log = logging.getLogger('jujugui')
```
Comment (hatched) [inline jujugui/views.py]: Yay logging!

**Comment by hatched** [comment]
:+1: with a trivial comment addition
No QA

**Comment by jaycee** [comment]
Thanks @hatched, @bac!

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2779/
Test PASSed.
